Dockerfile

```diff
@@ -1,9 +1,9 @@
 # Build the manager binary
-FROM quay.io/operator-framework/helm-operator:v1.33.0
+FROM --platform=linux/amd64 quay.io/operator-framework/helm-operator:v1.40.0
 
 ARG VERSION=${VERSION}
 ARG RELEASE_NUMBER=${RELEASE_NUMBER}
-ARG HELM_VERSION=3.15.3
+ARG HELM_VERSION=3.18.2
```

Comment:
@samiura could we use the existing helm version? Its possible that breaking changes could be causing this error in our tests

```
Error: "helm upgrade" requires 2 arguments

Usage:  helm upgrade [RELEASE] [CHART] [flags]
```
Either way, I'd recommend moving all the flags to the end, for example

`helm upgrade test-openshift sumologic/sumologic --install --version 4.13.0 ...`

Reply:
@rnishtala-sumo Good catch! This was more inline with their recommendation. We can probably use the current version and if needed change it if certification fails. Thanks a lot for pointing this out.

Reply:
I will make the changes as you pointed out either way because sooner or later we will get into this issue in the near future.